scripts/translator.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
+import * as fs from 'fs';
+import { sync as globSync } from 'glob';
+import { sync as mkdirpSync } from 'mkdirp';
+
+const filePattern = './build/messages/**/*.json';
+const outputLanguageDataDir = './src/languageProvider/locales/';
+
+let prevmessages = JSON.parse(fs.readFileSync('./src/languageProvider/locales/en_US.json', 'utf8'));
+let cnMessages = JSON.parse(fs.readFileSync('./src/languageProvider/locales/zh-Hans.json', 'utf8'));
+// Aggregates the default messages that were extracted from the app's
+// React components via the React Intl Babel plugin. The result
+// is a flat collection of `id: message` pairs for the app's default locale.
+let cnNew = {};
+let defaultMessages = globSync(filePattern)
+  .map((filename) => fs.readFileSync(filename, 'utf8'))
+  .map((file) => JSON.parse(file))
+  .reduce((collection, descriptors) => {
+    descriptors.forEach(({ id, defaultMessage }) => {
+      collection[id] = defaultMessage;
+      cnNew[id] = defaultMessage;
+      if(prevmessages.hasOwnProperty(id)){
+        if(collection[id]!==prevmessages[id]){
```

Comment:
lol can you add the `/script` folder when running `lint:fix` run script. this is hard to read

Reply:
is this command like `npm run lint:fix scripts/translator.js`

Reply:
oh looks like the lint command already does it on all files. I think you just didnt run `lint:fix`. please do so.